lib/model-builder.js

```diff
@@ -209,6 +211,11 @@ ModelBuilder.prototype.define = function defineClass(className, properties, sett
 
   util.inherits(ModelClass, ModelBaseClass);
 
+  // inherit validations
+  if (ModelBaseClass.validations && !settings.realmRequired) {
+    ModelClass.validations = traverse(ModelBaseClass.validations).clone();
```

Comment:
Really we use inheritance here to achieve mixin. For example,

```
var SubModel = class extends BaseModel {
  // Additional static properties
  // Additional static methods
  // Additional prototype properties
  // Additional prototype methods
}
```
With the code above, SubModel will inherit both static and prototype properties/methods from the BaseModel. But for our purpose, we want to clone the static properties so that the SubModel won't share them with BaseModel.